exasol_integration_test_docker_environment/test/test_test_env_reuse.py

```diff
@@ -0,0 +1,103 @@
+import os
+import unittest
+
+import docker
+import luigi
+
+from exasol_integration_test_docker_environment.lib.data.environment_type import EnvironmentType
+from exasol_integration_test_docker_environment.lib.test_environment.spawn_test_environment import SpawnTestEnvironment
+from exasol_integration_test_docker_environment.test.utils import process_spawn_utils
+from exasol_integration_test_docker_environment.cli.common import set_docker_repository_config
+from exasol_integration_test_docker_environment.test.utils import luigi_utils
+from exasol_integration_test_docker_environment.cli.options import test_environment_options
+
+
+class TestContainerReuseTest(unittest.TestCase):
+    '''
+    This test spawns a new test environment and, with parameters:
+    * reuse_database_setup=True,
+    * reuse_database=True,
+    * reuse_test_container=True
+    and verifies if the test data was populated to the docker db.
+    '''
+
+    def env_name(self):
+        return self.__class__.__name__.lower()
+
+    def setUp(self):
+        self._client = docker.from_env()
+        self._docker_repository_name = self.env_name()
+        print("docker_repository_name", self._docker_repository_name)
+        luigi_utils.clean(self._docker_repository_name)
```

Comment:
we might need to run the cleanup in it its own process

Reply:
Done

Reply:
As discussed offline, this is currently not feasible with current architecture.